realm/realm-library/src/main/cpp/jni_util/bcopy.hpp

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2017 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef REALM_BCOPY_H
+#define REALM_BCOPY_H
+
+#include <cstddef>
+
+namespace realm {
+namespace jni_util {
+
+void* my_bcopy(void *, const void *, std::size_t);
```

Comment:
`my_bcopy` sounds strange... just `__bcopy`? Also, no need to have this in an independent header since it is not supposed to be used for any other places, just `extern` it from the `hack.cpp`?